### PR TITLE
Sr postgres config2

### PR DESCRIPTION
**Pull request by simen-rekkedal**
## Description

change back to windows in the analyze step

## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Authentication/Program.cs [hunk 1]
```diff
@@ -416,19 +423,21 @@ void ConfigurePostgreSql()
...
-        string connectionString = string.Format(
-            builder.Configuration.GetValue<string>("PostgreSqlSettings:AdminConnectionString"),
-            builder.Configuration.GetValue<string>("PostgreSqlSettings:AuthenticationDbAdminPassword"));
+        //Yuniql Migration Scripts
+        ConsoleTraceService traceService = new() { IsDebugEnabled = true };
 
         string workspacePath = Path.Combine(Environment.CurrentDirectory, builder.Configuration.GetValue<string>("PostgreSqlSettings:WorkspacePath"));
         if (builder.Environment.IsDevelopment())
         {
+            string connectionString = string.Format(
```
Comment (acn-dgopa) [inline src/Authentication/Program.cs:432]: Just wondering that this is under if development check, do we have similar for other environments?

Comment (simen-rekkedal) [reply src/Authentication/Program.cs:432]: no, it's only for testing scripts against a local db during dev

Comment (simen-rekkedal) [reply src/Authentication/Program.cs:432]: we will simplify this in a later PR

File: src/Authentication/appsettings.Development.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
     "AuditLog": true,
     "SystemUser": true
   },
-  "PostgreSqlSettings": {
+  "PostgreSQLSettings": {
     "EnableDBConnection": true,
     "WorkspacePath": "Persistance/Migration",
     "AdminConnectionString": "Host=localhost;Port=5432;Username=postgres;Password={0};Database=local_authentication_testdb",
```
Comment (acn-dgopa) [inline src/Authentication/appsettings.Development.json:60]: better to have consistent names everywhere

File: src/Authentication/appsettings.json [hunk 3]
```diff
@@ -38,7 +38,7 @@
     "AuditLog": false,
     "SystemUser": true
   },
-  "PostgreSqlSettings": {
+  "PostgreSQLSettings": {
     "EnableDBConnection": true,
     "WorkspacePath": "Persistance/Migration",
     "AdminConnectionString": "Host=localhost;Port=5432;Username=platform_authentication_admin;Password={0};Database=local_authentication_testdb",
```
[the remark this answers is not in the document]
Comment (simen-rekkedal) [reply src/Authentication/appsettings.json]: yes, this is the name it has in the Keyvault, our local name was wrong

[the remark this answers is not in the document]
Comment (simen-rekkedal) [reply src/Authentication/appsettings.json]: oops, a typo with an extra s

**Review by acn-dgopa — APPROVED** [review]

**Pull request merged**